libraries/Microsoft.Bot.Builder.Azure/AzureBlobTranscriptStore.cs

```diff
@@ -74,7 +74,7 @@ public AzureBlobTranscriptStore(CloudStorageAccount storageAccount, string conta
             }, isThreadSafe: true);
         }
 
-        private Lazy<CloudBlobContainer> Container { get; set; }
+        public Lazy<CloudBlobContainer> Container { get; set; }
```

Comment:
why do you need to make this public?

Reply:
I didn't that was leftover from something I was trying